suite-native/home-graph/src/slice.ts

```diff
@@ -28,3 +28,9 @@ export const getSelectedTimeFrame = (state: SliceState) => state.appGraph.select
 
 export const { setSelectedTimeFrame } = appGraphSlice.actions;
 export const appGraphReducer = appGraphSlice.reducer;
+
+export const graphStore: Store = configureStore({
```

Comment:
Move this to the test directly, code that is used only for tests should never be placed alongside normal code